src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs

```diff
@@ -56,7 +55,57 @@ public class DefaultHtmlGenerator : IHtmlGenerator
             IModelMetadataProvider metadataProvider,
```

Comment:
Leaning toward `[Obsolete]`ing this constructor and filing an issue to remove it in 2.0. For one thing, it's only referenced explicitly twice (in tests).

Thoughts @Eilon and @rynowak?